R/PipeOpTaskSurvClassif.R

```diff
@@ -0,0 +1,151 @@
+#' @title PipeOpTaskSurvClassif
+#' @name mlr_pipeops_trafotask_survclassif
+#' @template param_pipelines
+#'
+#' @description
+#' Transform [TaskSurv] to [TaskClassif][mlr3::TaskClassif] by creating multiple
+#' interval observations for each subject based on `cut`, with a `ped_status` variable
+#' indicating whether an event occurred in each interval.
+#'
+#' @section Input and Output Channels:
+#' Input and output channels are inherited from [PipeOp][mlr3pipelines::PipeOp].
+#'
+#' The output is the input [TaskSurv] transformed to a [TaskClassif][mlr3::TaskClassif]
+#' as well as the transformed data during prediction.
```

Comment:
please write mroe details about what is the transformed data and Task (eg additional columns?) and if the user should be using these